start-site/src/main/java/io/spring/start/site/extension/dependency/testcontainers/TestcontainersProjectGenerationConfiguration.java

```diff
@@ -68,6 +69,20 @@ public ImplicitDependencyHelpDocumentCustomizer testcontainersHelpCustomizer(Bui
 
 	@Configuration(proxyBeanMethods = false)
 	@ConditionalOnPlatformVersion("3.1.0-RC1")
+	static class ServiceConnectionsConfiguration {
+
+		@Bean
+		ServiceConnections serviceConnections(ObjectProvider<ServiceConnectionsCustomizer> customizers) {
+			ServiceConnections serviceConnections = new ServiceConnections();
+			customizers.orderedStream().forEach((customizer) -> customizer.customize(serviceConnections));
+			return serviceConnections;
+		}
+
+	}
+
+	@Configuration(proxyBeanMethods = false)
+	@ConditionalOnPlatformVersion("3.1.0-RC1")
```

Comment:
Would it be worth having one class that's `@ConditionalOnPlatformVersion("3.1.0-RC1")`? It could define the `ServiceConnections` bean and then have `SpringBootSupportConfiguration` and `TestApplicationConfiguration` as inner-classes.

Reply:
Good idea, thanks. Done.